src/stores/Annotation/Annotation.js

```diff
@@ -746,7 +756,7 @@ export const Annotation = types
         results: [result],
       };
 
-      const area = self.areas.put(areaRaw);
+      const area = self?.areas?.put(areaRaw);
```

Comment:
This change makes the below change no longer safer. `area` can now be undefined, so we will need to make sure the below checks `!area.classification` uses `area && !area.classification`.

Reply:
Similarily the line below it, maybe just early return `if (!area) return`.
```suggestion
      const area = self?.areas?.put(areaRaw);
      if (!area) return;
```

Reply:
Why do we need this safe reference? `self` is definitely defined and `self.areas` is defined mst required property.

Reply:
discussed it — weird mst bug possibly because of production build, so leave it as is